examples/concertbot/data/nlu.yaml

```diff
@@ -0,0 +1,39 @@
+nlu:
+  - intent: greet
+    examples: |
+      hi
```

Comment:
can you add the `-` prefix here? Like in Arjaans PR: https://github.com/RasaHQ/rasa/pull/6229/files#diff-ed593145bcc162429c6f89a64f2fcc0dR5